### DISCUSSION TITLE
ID Number changes per User on Mac OS

### DISCUSSION THREAD

**Discussion by PinataPost** [rustdesk/rustdesk#7096] [Q&A]
Hello, I wanted to know if this is an expected behavior. When I log out one user, and in as another, the ID changes, which makes it difficult to use as an IT support tool. 

Is this expected?  Is there a way to keep the ID / Pass consistent?

**Comment by rustdesk** [comment]
Done this?
![image](https://github.com/rustdesk/rustdesk/assets/71636191/d769c9f9-137b-4c4a-bf1b-8dfd160104f1)

**Reply by PinataPost** [reply]
Thanks for the quick reply. I just tested it on another machine, and after installing the service the ID does stay the same.

I think what happened is that I logged into another user before installing the it as a service.  I will recreate the user and try again.
